services/mail_service.js

```diff
@@ -0,0 +1,57 @@
+const fetch = require("node-fetch")
+
+const serviceUrl = "https://email-service-estella.herokuapp.com/email"
+const selfUrl = "https://meeting-organizer-estella.herokuapp.com/"
+
+const sendMail = async (body) => {
+    const result = await fetch(serviceUrl, {
+        method: "POST",
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify(body)
+    })
+    return result.status !== 200
+}
+
+const mailService = {
+    addMoreSlotsHosts: async (host, hosts, organizer, guest) => {
+        return await sendMail({
+            subject: "Add more time slots",
+            sender_email: organizer,
+            sender_name: organizer,
+            receiver: host.email,
+            content: `Hi
+                Add more time slots for meeting with ${guest}, because there is no slot where all of you can attend. 
+                You can do this at this link: ${selfUrl + "host/" + host.uuid}.
+                You can contact with other meeting participants: ${hosts.join(", ")}
+                `
+        })
+    },
+
+    askForMoreSlots: async (guest, hosts) => {
+        const promises = hosts.map(async host => await sendMail({
+            subject: "Guest asked for adding more slots for interview",
+            sender_email: guest,
+            sender_name: guest,
+            receiver: host,
+            content: `Hi 
```

Comment:
po Hi powinien lecieć przecinek i nie wiem, czy Hi to najlepszy sposób rozpoczęcia oficjalnego maila do deva, choć w sumie wszyscy się tak w it witają, to może przesadzam.